pykern/pkcli/github.py

```diff
@@ -28,12 +28,15 @@
 _GITHUB_HOST = "github.com"
 _GITHUB_URI = "https://" + _GITHUB_HOST
 _GITHUB_API = "https://api." + _GITHUB_HOST
-_WIKI_ERROR_OK = (
-    r"fatal: repository 'https://github.com/radiasoft/.*.wiki.git/' not found"
-)
+_WIKI_ERROR_OK = r"fatal: repository 'https://github.com/[\w\-\.\_]+/[\w\-\.\_]+.wiki.git/' not found"
```

Comment:
[\w matches underscore](https://docs.python.org/3/library/re.html). I think add '/' to the regex and eliminate the need for both of them. Something like:

```python
_WIKI_ERROR_OK = r"fatal: repository 'https://github.com/[-/\w\.]+.wiki.git/' not found"
```